components/uuf-core/src/test/java/org/wso2/carbon/uuf/core/APITest.java

```diff
@@ -70,6 +75,8 @@ public void testSendRedirect() {
 
     @Test
     public void testCreateSession() {
```

Comment:
IMO now this test is no longer required as the `API.createSession()` method doesn't contain any logic. It just calls the `SessionHandler.createSession()` method. May be we should write a test class for `InMemorySessionManager`.